src/aap_eda/core/models/activation.py

```diff
@@ -117,3 +127,9 @@ class Meta:
         default=None,
         help_text="Name of the kubernetes service",
     )
+
+    def save(self, *args, **kwargs):
```

Comment:
https://docs.djangoproject.com/en/5.0/topics/db/queries/#updating-multiple-objects-at-once

Reply:
Using filter/update instead of get/save is how django scales to large numbers of rows and bulk operations.   Putting logic in save() eliminates the possibility of doing bulk operations.

Reply:
Put up this PR to change this following @mkanoor 's suggestion in earlier comments:
https://github.com/ansible/eda-server/pull/831